docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst

```diff
@@ -110,7 +110,7 @@ We will add the following code to the file.
   - name: Test for new_option, create new user WITHOUT the attribute
     postgresql_user:
       name: test_user
-      add_attribute: no
+      add_attribute: false
```

Comment:
`add_attribute` is not a valid parameter for `postgresql_user` (atleast I was not able to find it in the docs). Can you please remove it/verify if it exists?

Reply:
cc @Andersson007

Reply:
i also don't see it, should be removed then, @Akasurde thanks

Reply:
@Akasurde  So you want me to remove  `add_attribute` here for the `postgresql_user`  right?

Reply:
@Akasurde @Andersson007 - is it as simple as removing that one line, or does that impact the rest of the example? We can make the suggested edits to remove in this PR, but please review the full example and let us know if more needs to change (besides FQCN).

Reply:
Let us tackle this in separate PR since the whole example is talking about creating a user without attributes.